### DISCUSSION TITLE
BMW電動車貼上「可變色電子紙」 美國消費電子展吸睛登場！

### DISCUSSION THREAD

**Discussion by moonnight3503** [1101-GS4719-C/Python-Community-GS4719-C#1034] [General]
昨天看新聞剛好看到，BMW出了一款電動變色的車款，利用在車子周圍貼上可變色電子紙，最多可變換32種不同顏色，並透過程式編輯讓車身出現色彩動態變化，主打低耗能、又酷炫，雖然他是很炫沒錯，但身為平民老百姓的我，我應該是不會想要去買這種出款啦，而且雖然低耗能但感覺還是很吃電，我可能比較prefer買一款防撞性能高一點的車款比較實際哈哈。

連結:https://news.ttv.com.tw/news/11201060032200L
![image](https://user-images.githubusercontent.com/113499537/211135530-dc192a5a-0a2e-47eb-b18c-137225809a5c.png)

**Comment by Valentinaa0214** [comment]
超級酷炫！感覺每天可以隨著心情變換不同顏色的車體外觀。
每次變換顏色都像換了一台新車一樣！
是我有錢會買😀😀😀

**Comment by Jasper525** [comment]
如果電子紙裡面是rbg led燈的話，還有一個好處是藉由發光性，提升車子在低光源環境的可視度，增加夜間交通安全😂

**Comment by manipulateed** [comment]
電動車的外來看來將會非常多元，而且功能將很多變，誰能走到最後成為霸主都說不一定，也有可能是彼此合作，共享技術，共同繁榮呢。

**Comment by dabbie109303501** [comment]
覺得電子紙的產品很酷 所以特地找了一下原理
原來是透過夾層中的三原色粒子因電場的改變 調換順序 以產生不同的顏色
![image](https://user-images.githubusercontent.com/75252877/211144408-96cdb015-19f7-41e9-bb60-e8d319b1b1c6.png)
![image](https://user-images.githubusercontent.com/75252877/211144424-0394fd4e-4d91-4c1f-be83-06707d0c1781.png)

來源:https://tw.eink.com/electronic-ink.html

**Comment by Kotori0501** [comment]
我也不太會選這款車，因為我首選的主要條件真的是要防撞功能比較好的，路上真的很容易發生車禍，所以這是我首要條件，酷歸酷，但真的要耐撞，除非我非常有錢可以買一台在家擺放，我才會去買。

**Comment by Twenty0522** [comment]
這款車可以非常高調，但自己坐在車內時無法看到，所以不會想購買。
